.github/workflows/macos.yml

```diff
@@ -233,6 +233,11 @@ jobs:
           echo '::group::raw'; cat bld/lib/curl_config.h || true; echo '::endgroup::'
           cat bld/lib/curl_config.h | grep -F '#define' | sort || true
 
+      - name: 'build-cert'
+        run: |
+          make -C bld/tests/certs V=1 clean-certs
+          make -C bld/tests/certs V=1 build-certs -j1
```

Comment:
What is the reason for rebuild the test certs for every test?

Reply:
removed